sites/outsite.nl/components/JoinOptions.vue

```diff
@@ -56,14 +56,14 @@ en:
 nl:
   title: Kennismaken met <strong>Outsite</strong>?
   kmg:
-    title: Kennismakingsgroepen (KMG)
+    title: "Kennismakingsgroepen (KMG)"
```

Comment:
Why do we need the `"` here (and below as well)? :)

Reply:
I initially got several errors when trying to render this page, and it worked after I put quotes around all the strings with special characters (like `(`) in them, so I figured those were related :p